app/controllers/image_uploads_controller.rb

```diff
@@ -28,18 +26,28 @@ def create
       return
     end
 
-    cloudinary_link(uploader)
+    cloudinary_link(uploaders)
   end
 
-  def cloudinary_link(uploader)
-    link = if params[:wrap_cloudinary]
-             ApplicationController.helpers.cloud_cover_url(uploader.url)
-           else
-             uploader.url
-           end
-
+  def cloudinary_link(uploaders)
+    links = if params[:wrap_cloudinary]
+              [ApplicationController.helpers.cloud_cover_url(uploaders[0].url)]
+            else
+              uploaders.map(&:url)
+            end
     respond_to do |format|
-      format.json { render json: { link: link }, status: :ok }
+      format.json { render json: { links: links }, status: :ok }
+    end
+  end
+
+  private
+
+  def image_upload(images)
```

Comment:
This is cool, because in the future we could also implement parallel upload to make it faster!